dspace-api/src/main/java/org/dspace/content/service/clarin/ClarinItemService.java

```diff
@@ -90,4 +90,15 @@ public interface ClarinItemService {
      */
     void updateItemFilesMetadata(Context context, Bitstream bit) throws SQLException;
 
+    /**
+     * Update item's metadata about its dates (dc.date.issued, local.approximateDate.issued).
+     * If the local.approximateDate.issued has a value like 'cca 1938 - 1945' or something else,
+     * then dc.date.issued = 0000.
+     * If the local.approximateDate.issued has a value like '1938, 1945, 2022', then dc.date.issued = 2022.
```

Comment:
This doc comment is really not clear at all. And why would this be done?

Reply:
Check the issue in the description.

Reply:
add to second line (L95) 
if the l.aD.i has any approximate value, e.g. 'cca 1938 - 1945'  or 'approx. 1995' or similar, use 0000

add to last line (L97) 
if l.aD.i has several values, e.g. 1993,1918,2021 use the last one: d.d.i = 2021

Reply:
Done